docs/source/graph-construction/gs-processing/example.rst

```diff
@@ -1,41 +1,50 @@
 .. _distributed_construction_example:
 
-GraphStorm Processing Example
-=============================
+GraphStorm Distributed Construction Example
+===========================================
 
-To demonstrate how to use the library locally we will
+To demonstrate how to use distributed graph construction locally we will
 use the same example data as we use in our
 unit tests, which you can find in the project's repository,
 under ``graphstorm/graphstorm-processing/tests/resources/small_heterogeneous_graph``.
 
 Install example dependencies
 ----------------------------
 
-To run the local example you will need to install the GSProcessing
+To run the local example you will need to install the GSProcessing and GraphStorm
 library to your Python environment, and you'll need to clone the
-GraphStorm repository to get access to the data.
+GraphStorm repository to get access to the data, and dgl tool for GPartition.
 
 Follow the :ref:`gsp-installation-ref` guide to install the GSProcessing library.
 
-You can clone the repository using
+To install the GPartition library:
 
 .. code-block:: bash
 
+    pip install graphstorm
+    pip install pydantic
+    pip install torch==2.1.0 --index-url https://download.pytorch.org/whl/cpu
+    pip install dgl==1.1.3 -f https://data.dgl.ai/wheels-internal/repo.html
     git clone https://github.com/awslabs/graphstorm.git
+    cd graphstorm
+    git clone --branch v1.1.3 https://github.com/dmlc/dgl.git
```

Comment:
Not refer to the guide we have in the quick-start, because we need to install the tool script and pydantic, which are not required in the main gsf repo.